Add ldap server example (#105)
Mirantis/mke-docs: content/docs/tutorials/k0s-in-aws/terraform-scenario.md

3. Edit the `apiServer.externalAddress` in the configuration file

   ```sh
   terraform output -raw lb_dns_name | { read lb; yq -i ".apiServer.externalAddress = \"$lb\"" mke4.yaml; }
   ```

>    If you do not have the `yq` tool installed, edit the `mke4.yaml` file manually
>    setting `apiServer.externalAddress` to the output of the `terraform output -raw lb_dns_name` command.

Reviewer: ```suggestion
   If the `yq` tool is not installed, edit the `mke4.yaml` file to set `apiServer.externalAddress` as the output of the `terraform output -raw lb_dns_name` command.
```

Author: This looks like some kind of rebase error. I'm going to leave this change out since it doesn't relate to my PR and I will remove this change from my commit
